project/src/components/poster/poster.tsx

```diff
@@ -0,0 +1,23 @@
+type FilmPosterProps = {
+  posterSrc: string;
+  posterTitle: string;
+  className?: string;
+}
+
+function Poster(props: FilmPosterProps): JSX.Element {
+
+  const {posterSrc, posterTitle, className} = props;
+
+  return (
+    <div className={`film-card__poster ${className || ''}`}>
```

Comment:
```suggestion
    <div className={`film-card__poster ${className ?? ""}`}>
```

ну а вообще для работы с классами есть библиотека classnames, шикарная штука - используй.